cmd/podman/ps.go

```diff
@@ -324,7 +324,7 @@ func generateContainerFilterFuncs(filter, filterValue string, runtime *libpod.Ru
 			return nil, errors.Wrapf(err, "exited code out of range %q", filterValue)
 		}
 		return func(c *libpod.Container) bool {
-			ec, err := c.ExitCode()
+			ec, _, err := c.ExitCode()
```

Comment:
Shold this be ignored here?

Reply:
This is just for filtering containers - let me check what Docker does so we can match

Reply:
It looks like we ignore any containers that have not exited - fixing